src/db/migrations/20190408115030-add-validation-columns-users.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await true
```

Comment:
What is `await true` for? (this question applies to all such cases)

Reply:
Whoops, missed that in cleanup

Reply:
Cleaned up. Closing.